aquadoggo/src/network/peers/handler.rs

```diff
@@ -237,33 +227,11 @@ impl ConnectionHandler for Handler {
         }
     }
 
-    fn connection_keep_alive(&self) -> KeepAlive {
```

Comment:
Custom implementation of `connection_keep_alive` is no longer required as connections are now kept alive unless all streams are closed, and for us that never happens unless a critical error occurs.